surface/include/pcl/surface/marching_cubes_hoppe.h

```diff
@@ -71,15 +72,35 @@ namespace pcl
 
 
       /** \brief Constructor. */
-      MarchingCubesHoppe ();
+      MarchingCubesHoppe (const float percentage_extend_grid = 0.0f,
+                          const float iso_level = 0.0f,
+                          const float dist_ignore = -1.0f) :
+        MarchingCubes<PointNT> (percentage_extend_grid, iso_level),
```

Comment:
Since you're more knowledgeable about this algorithm than I am, does the order in which these parameters appear respect the convention of "more likely to be changed" to "less likely to be changed" (from the default values)?

(I ask the same below)